actions/app.go

```diff
@@ -44,13 +47,19 @@ func App() *buffalo.App {
 	mem := memory.New()
 	storageReader := mem
 	storageWriter := mem
+	mgoStore := mongo.NewMongoUserStore("127.0.0.1:27017")
+	err := mgoStore.Connect()
+	if err != nil {
+		panic(err)
```

Comment:
Are we doing any error handling yet elsewhere? Or is panic the norm?

Reply:
in cases where the application can't start (like because the DB is unavailable) a panic is appropriate.